modules/installation-gcp-permissions.adoc

```diff
@@ -30,31 +31,39 @@ When you attach the `Owner` role to the service account that you create, you gra
 .Required roles for creating network resources during installation
 * DNS Administrator
 
-.Required roles for using passthrough credentials mode
+.Required roles for using the Cloud Credential Operator in passthrough mode
 * Compute Load Balancer Admin
-* IAM Role Viewer
 
 ifdef::template[]
 .Required roles for user-provisioned GCP infrastructure
 * Deployment Manager Editor
 endif::template[]
 
-The roles are applied to the service accounts that the control plane and compute machines use:
+The service accounts that the control plane and compute machines use require the following roles and permissions:
 
-.GCP service account permissions
-[cols="2a,2a",options="header"]
+.GCP service account roles and permissions
```

Comment:
This lists in this table were already out of sync with the list for mint mode. Not sure all of these belong here, or what the control plane vs compute distinction is doing exactly

Reply:
We probably don't need to touch these; I believe they are simply listing the roles given to the control plane and compute machine service accounts, which are created by the installer.

https://github.com/openshift/installer/blob/1bad952b455480d8fe6523fa0e39e0191673040f/data/data/gcp/cluster/master/main.tf#L5-L37

https://github.com/openshift/installer/blob/1bad952b455480d8fe6523fa0e39e0191673040f/data/data/gcp/cluster/iam/main.tf#L5-L20